_articles/getting-paid.md

```diff
@@ -145,6 +154,17 @@ Some popular projects, like [npm](https://github.com/npm/npm) and [Docker](https
 
 Some software foundations and companies offer grants for open source work. Sometimes, grants can be paid out to individuals without setting up a legal entity for the project.
 
+Few organisation enabling grants for open source projects are:
```

Comment:
Items from this list should be added to the following list, in the same style, with

PROJECT was funded by FUNDER

Probably not every item. Even adding one really solid example in format above would be great. If there aren't any, that's OK too!

As [mentioned in the issue ](https://github.com/github/opensource.guide/issues/1058#issuecomment-521050643) I don't think adding a plain list to the article is consistent with the approach of the guides overall.

Reply:
I respectfully disagree. 

Instead of giving example of organisations and forcing the readers for establishing a relation or a sample of an project/org who received a grant, if the documentation is proper, it should guide for options available. 

There are numerous projects, orgs that has received funding at a point of time. We should not force the readers for establishing any relation with any projects. They are open for just reading the funding/grants options available for them and going ahead. 

If you disagree, feel free closing the PR. Your suggestion is against the stance I take while writing docs for readers. Thank you.

Reply:
> I would instead delete the PROJECT was funded by FUNDER list, I find that ugly and useless.

I admit, I do too -- indeed I find all user/individual call outs and quotes useless -- but the guides aren't written for me, and it's the style used throughout the guides. I think the idea is to keep the guides human and relatable -- they aren't straight up documentation. I tried to add my understanding of the approach we've used to lists in #1121. 

I'm going to close this as I don't think it's a good fit for the guides as they are now, though I really appreciate the PR, explanation, and respectful disagreement. Thank you!